.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:
+
+- [ ] Branch rebased on latest master `git pull --rebase upstream master`
+- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
+- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
+- [ ] Commit message includes `Fixes #<id>` where `<id>` is the associated MantisBT issue id if one exists
+
+And please remember to update the Wiki with the features added or changed once this PR once it is merged.
```

Comment:
Interesting.. didn't know this feature existed! Will oyu add @luzpaz 's suggestions @bblacey ? Then I'll wait before merging. Also maybe
>  [ ] Commit message includes `Fixes #<id>` where `<id>` is the associated MantisBT issue id if one exists

 could become:
>  [ ] Commit message includes `issue #<id>`  or `fixes #<id>` where `<id>` is the associated MantisBT issue id if one exists

because issue #id won't close the ticket, it's useful too...